app/build.gradle

```diff
@@ -27,7 +27,9 @@ dependencies {
     implementation "com.android.support:appcompat-v7:$rootProject.supportLibraryVersion"
     implementation "com.android.support:design:$rootProject.supportLibraryVersion"
     implementation "com.android.support.constraint:constraint-layout:$rootProject.constraintLayoutVersion"
+    implementation 'com.android.support:preference-v7:26.1.0'
```

Comment:
Use the same library version across the whole project. Change this to;
`implementation "com.android.support:preference-v7:$rootProject.supportLibraryVersion"`